README.md

```diff
@@ -87,30 +90,46 @@ act -v
   -W, --workflows string       path to workflow files (default "./.github/workflows/")
 ```
 
+# Known Issues
+
+MODULE_NOT_FOUND during `docker cp` command
```

Comment:
Thanks @gjvpaet - can you link to the open issue (#228) in the PR?

Reply:
Done @cplee